bridge_ui/src/utils/consts.ts

```diff
@@ -243,7 +271,8 @@ export const TERRA_HOST =
         name: "testnet",
       }
     : {
-        URL: "http://localhost:1317",
+        // URL: "http://localhost:1317",
+        URL: "http://188.166.208.240:1317",  // TODO: change back
         chainID: "columbus-5",
         name: "localterra",
       };
```

Comment:
Add `export const KARURA_HOST_WS` and `export const KARURA_HOST_RPC` here instead of hardcode.